astropy/io/ascii/ui.py

```diff
@@ -32,7 +33,9 @@
 from ...table import Table
 from ...utils.data import get_readable_fileobj
 from ...extern import six
+from ...utils.exceptions import AstropyDeprecationWarning
 
+_diffkeyArray=[]
```

Comment:
There's no reason for this to be a global variable. That said, see https://github.com/astropy/astropy/pull/4511/files#r50255334